docs/source/paper.md

```diff
@@ -28,9 +28,13 @@ bibliography: paper.bib
+On the software level these problems are perhaps best addressed by using messaging brokers that take responsibility for guaranteeing the durability and atomicity of messages and often enable event-based communication.  Indeed, solutions such as [RabbitMQ](https://www.rabbitmq.com/) see widespread adoption in industry, however, adoption in academia has been more limited, with home-made queue data structures, race condition susceptible locks and polling based solutions being commonplace. This is likely due to message brokers typically having complex APIs (which reflect the non-trivial nature of the underlying protocol) as well as the lack of familiarity with event-based programming in general within the community.  [KiwiPy](https://kiwipy.readthedocs.io/en/latest/) was designed specifically to address both these issues, by providing a tool that allows building robust, event-based systems with an interface that is as simple as possible.
 
-KiwiPy provides three main message types to the user, task queues, Remote Procedure Calls (RPCs), and, broadcasts. These are all exposed via one class called the `Communicator` which can be trivially constructed by providing a URI string pointing to the RabbitMQ server. This starkly contrasts with the several steps that are required to establish a connection to perform each of the message types listed above using [standard libraries](https://www.rabbitmq.com/getstarted.html). Furthermore, by default, kiwiPy creates a separate communication thread that the user never sees, allowing them to interact with the communicator using familiar Python syntax, without the need to be familiar with either coroutines or multithreading.  This has the additional advantage that kiwiPy will maintain heartbeats (a periodic check to make sure the connection is still alive) with the server whilst the user code can be doing other things. Heartbeats are an essential part of RabbitMQ's fault tolerance whereby two missed checks will automatically trigger the message to be requeued to be picked up by another client.
+KiwiPy provides three main message types to the user: task queues, Remote Procedure Calls (RPCs), and, broadcasts.
+These are all exposed via one class called the `Communicator` which can be trivially constructed by providing a URI string pointing to the RabbitMQ server.
+This is in contrasts with the several steps that are required to establish a connection to perform each of the communication types listed above using [Pika](https://pika.readthedocs.io/en/stable/), the standard library used in RabbitMQ [tutorials](https://www.rabbitmq.com/getstarted.html).
+ A more in depth of the two can be found in our [documentation](https://kiwipy.readthedocs.io/en/latest/).
```

Comment:
```suggestion
 A more in depth comparison of the two can be found in our [documentation](https://kiwipy.readthedocs.io/en/latest/pika-comparison).
```
Not sure if I put the correct URL but I think it would be better to link straight to the new comparison page

Reply:
I thought to do that but I deliberately held off because I want the freedom to restructure the docs in the future if need be (while the paper will be static).  I see two options:

1) Link it to a particular `kiwi` version on RTD and make sure that one is kept in-perpetuity, or,
2) Leave it as is.

Do you have a preference?

Reply:
Ah I see, fair point. In that case, for simplicity and consistency I think we should keep what you have now.